libpod/container_log.go

```diff
@@ -90,10 +104,11 @@ func (c *Container) readFromLogFile(options *logs.LogOptions, logChannel chan *l
 						logrus.Error(err)
 					}
 					break
+				} else {
+					<-eventChannel
 				}
```

Comment:
```suggestion
				} 
				<-eventChannel
```

Reply:
@QiWang19 The logic should be the same before and after your change. When it hits the first "IF" block it will break out of the loop. Else it will wait for an event.

Reply:
@QiWang19 If this is the style of condition statements you all prefer let me know; I will make the change. I think this is a code styling preference suggestion only. Please correct me if I am in error.